tests/integration/default-properties-test.js

```diff
@@ -76,5 +76,14 @@ if (require.has('@ember/test-helpers')) {
       assert.equal(page.dummy.isVisible(), 'isVisible');
       assert.equal(page.dummy.text(), 'text');
     });
+
+    test('allows empty create', async function(assert) {
+      let page = create();
+      page.setContext(this);
```

Comment:
I think `setContext` should be removed, cause it only needed for `moduleForComponent` test

Reply:
removed.